fabric-networking-api-v1/src/testmod/java/net/fabricmc/fabric/test/networking/play/NetworkingPlayPacketTest.java

```diff
@@ -117,6 +139,13 @@ public void onInitialize() {
 				}
 			}
 		});
+
+		ServerLoginConnectionEvents.INIT.register((handler, server) -> {
+			((ServerCookieStore) handler).getCookie(new Identifier("fabric:test")).whenComplete((data, throwable) -> {
```

Comment:
Check if the player was transfered, and then get the cookie.

Reply:
Done